std_msgs/Header header

int8 x			# x-axis [%]
int8 y			# y-axis [%]
int8 z			# z-axis [%]
int8 yaw 		# yaw-axis [%]
int8 camera		# camera-axis [%]
int8 zoom		# zoom-axis [%]
bool RTL 		# RTL (front top) button {True,False}
bool takeoff_land 	# takeoff_land (front bottom) button {True,False}
bool reset_camera 	# reset_camera (back left) button {True,False}
bool reset_zoom 	# reset_zoom (back right) button {True,False}
